docs/reference/tab-widgets/troubleshooting/data/increase-node-capacity.asciidoc

```diff
@@ -0,0 +1,299 @@
+----
+GET /my-index-000001/_settings/index.routing.allocation.include._tier_preference?flat_settings
+----
+// TEST[continued]
+
+
+The reponse will look like this:
+
+[source,console-result]
+----
+{
+  "my-index-000001": {
+    "settings": {
+      "index.routing.allocation.include._tier_preference": "data_warm,data_hot" <1>
+    }
+  }
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+
+
+<1> Represents a comma separated list of data tier node roles this index is allowed
+to be allocated on, the first one in the list being the one with the higher priority
+i.e. the tier the index is targeting.
+e.g. in this example the tier preference is `data_warm,data_hot` so the index is
+targeting the `warm` tier and more nodes with the `data_warm` role are needed in
+the {es} cluster.
+
+
+Alternatively, if adding more nodes to the {es} cluster is not desired,
+inspect the <<dynamic-index-number-of-replicas,`index.number_of_replicas`>> index setting and
+decrease the configured value:
+
+
+. Inspect the <<dynamic-index-number-of-replicas,`index.number_of_replicas`>> index setting for the
+index with unassigned replica shards:
++
+[source,console]
+----
+GET /my-index-000001/_settings/index.number_of_replicas
+----
++
+The response will look like this:
++
+[source,console-result]
+----
+{
+  "my-index-000001" : {
+    "settings" : {
+      "index" : {
+        "number_of_replicas" : "2" <1>
+      }
+    }
+  }
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only as don't want to change a cluster-wide setting]
+
++
+<1> Represents the currently configured value for the number of replica shards
+required for the index
+
+. Use the `_cat/nodes` API to find the number of nodes in the target tier:
++
+[source,console]
+----
+GET /_cat/nodes?h=node.role
+----
+// TEST[continued]
++
+The response will look like this, containing one row per node:
++
+[source,console-result]
+----
+himrst
+mv
+himrst
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
++
+You can count the rows containing the letter representing the target tier to know how many nodes you have. See
+<<cat-nodes-api-query-params>> for details. The example above has two rows containing `h`, so there are two
+nodes in the hot tier.
++
+. <<indices-update-settings, Decrease>> the
+value for the total number of replica shards required for this index. The new value needs to be less
+than the number of nodes found above minus one. Since the example above found 2 nodes in the hot tier,
+the maximum value for `index.number_of_replicas` is 1.
++
+[source,console]
+----
+PUT /my-index-000001/_settings
+{
+  "index" : {
+    "number_of_replicas" : 1 <1>
+  }
+}
+----
+// TEST[continued]
+
++
+<1> The new value for the `index.number_of_replicas` index configuration
+is decreased from the previous value of `2` to `1`. It can be set as low as 0 but
+configuring it to 0 may lead to temporary availability loss during node restarts
```

Comment:
```suggestion
is decreased from the previous value of `2` to `1`. It can be set as low as 0 
but configuring it to 0 for indices other than <<searchable-snapshots,
searchable snapshot indices>> may lead to temporary availability loss during 
node restarts
```